### PR TITLE
fix logging message in `executeCommand` event

### PR DESCRIPTION
**Pull request by tunnckoCore**
suggested pr by @ngoldman lol

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ function App (el) {
         self.emit('leaveChannel', self.data.activeChannel.name)
         break
       default:
-        console.log('Unrecognized /command: ' + command)
+        console.log('Unrecognized command: ' + commandStr)
```
Comment (ngoldman) [inline app.js]: should still be `command`, no? `commandStr` is unprocessed by https://github.com/tunnckoCore/friends/blob/master/app.js#L232-L234

Comment (ngoldman) [inline app.js]: I'll merge and mess with it a bit

**Comment by bcomnes** [comment]
Good catch @tunnckoCore

File: app.js (hunk 1 above)
Comment (tunnckoCore) [inline app.js]: no, exactly. should be `command` to have output
```
Unrecognized command: /hello
```
that's was the idea i mentioned

**Pull request merged**